app/assets/stylesheets/active_admin/components/_dropdown_menu.scss

```diff
@@ -1,151 +1,45 @@
-.dropdown_menu {
-  display: inline;
-
-  .dropdown_menu_button {
-    @include light-button;
-    position: relative;
-    padding-right: 22px !important;
-    cursor: pointer;
-
-    &:before {
-      content: ' ';
-      position: absolute;
-      width: 0;
-      height: 0;
-      border-width: 3px 3px 0;
-      border-style: solid;
-      border-color: #FFF transparent;
-      right: 12px;
-      top: 45%;
-    }
-
-    &:after {
-      content: ' ';
-      position: absolute;
-      width: 0;
-      height: 0;
-      border-width: 3px 3px 0;
-      border-style: solid;
-      border-color: #777 transparent;
-      right: 12px;
-      top: 45%;
-    }
-  }
-
-  .dropdown_menu_nipple {
-
-    // The nipple's border
-    content: "";
-    position: absolute;
-    top: -6px;
-    display: block;
-    width: 0;
-    height: 0;
-    border-width: 0 6px 6px;
-    border-style: solid;
-    border-color: darken($primary-color, 4%) transparent;
-    z-index: 100;
-
-    // The nipple's inner shadow
-
-    &:before {
-      content: ' ';
-      position: absolute;
-      width: 0;
-      height: 0;
-      border-width: 0 5px 5px;
-      border-style: solid;
-      border-color: lighten($primary-color, 15%) transparent;
-      left: -5px;
-      top: 1px;
-    }
-
-    // The nipple's background color
-
-    &:after {
-      content: ' ';
-      position: absolute;
-      width: 0;
-      height: 0;
-      border-width: 0 5px 5px;
-      border-style: solid;
-      border-color: lighten($primary-color, 4%) transparent;
-      left: -5px;
-      top: 2px;
-    }
-  }
+.caret {
+  border-left: $caret-width-base solid transparent;
+  border-right: $caret-width-base solid transparent;
```

Comment:
Color literals like `transparent` should only be used in variable declarations; they should be referred to via variable everywhere else.